app/src/Filters.jsx

```diff
@@ -3,14 +3,15 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faFilter } from '@fortawesome/free-solid-svg-icons';
 
 import { AppContext, getFilterParams } from './AppContext';
+import { pathExistsSync } from 'fs-extra';
```

Comment:
I can't think of a reason for this import to be here?

Reply:
Yes you're right. i'm not sure of it, maybe it was created when i was defining function. i'll remove it.